src/gui/folderman.h

```diff
@@ -134,8 +145,11 @@ class FolderMan : public QObject
     /// Produce text for use in the tray tooltip
     static QString trayTooltipStatusString(SyncResult::Status syncStatus, bool hasUnresolvedConflicts, bool paused);
 
-    /// Compute status summarizing multiple folders
-    static void trayOverallStatus(const QList<Folder *> &folders, SyncResult::Status &status, bool &unresolvedConflicts, QDateTime&lastSyncDone);
+    /**
+     * Compute status summarizing multiple folders
+     * @return tuple containing folders, status, unresolvedConflicts and lastSyncDone
+     */
+    static TrayOverallStatusResult trayOverallStatus(QList<Folder *> folders);
```

Comment:
can we use a const & for the list?

Reply:
Yeah, sure, my bad. Although copying a list of pointers isn't that big of a deal, it's of course better to use references where possible.